src/components/List.js

```diff
@@ -43,6 +46,21 @@ const List = ({ firestore }) => {
     }
   };
 
+  const handleDelete = id => {
+    setIdToDelete(id);
+    setModalDisplay(true);
```

Comment:
Curious why this is needed here. Isn’t the modal already showing?

Reply:
What happens here is:

- The delete item button is clicked
    - That sets the display mode to true, and the modal is displayed
    - It also saves the id of item to be deleted into state
- The 'yes' button in the modal is clicked
    - an async delete request is sent to firestore
    - when that completes, the display mode is set to false and the modal goes away

Reply:
Ah, I get it now! Thanks!